bundles/quote/src/FondOfKudu/Zed/Quote/Business/SuccessOrderQuote/SuccessOrderQuoteDeleter.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace FondOfKudu\Zed\Quote\Business\SuccessOrderQuote;
+
+use DateInterval;
+use DateTime;
+use FondOfKudu\Zed\Quote\Persistence\QuoteRepositoryInterface;
+use FondOfKudu\Zed\Quote\QuoteConfig;
+use Generated\Shared\Transfer\QuoteCollectionTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Zed\Kernel\Persistence\EntityManager\TransactionTrait;
+use Spryker\Zed\Quote\Persistence\QuoteEntityManagerInterface;
+
+class SuccessOrderQuoteDeleter implements SuccessOrderQuoteDeleterInterface
+{
+    use TransactionTrait;
+
+    /**
+     * @var int
+     */
+    protected const BATCH_SIZE_LIMIT = 200;
```

Comment:
maybe configurable?

Reply:
I thought of that too, but I orientated myself on Spryker. But I also think it would be better if we made it configurable.